### PR TITLE
feat: Types for MiniTooltip

### PR DESCRIPTION
**Pull request by Golodhros**
## Description

Types for the mini tooltip

## Motivation and Context

Completing our types

## How Has This Been Tested?

Manually on the sandbox

## Screenshots (if appropriate):
Fullscreen_10_19_20__10_16_PM

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Build is successful
- [ ] Updated the documentation
- [ ] Added tests to cover changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: src/typings/charts/mini-tooltip.d.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+type ChartSize = [
+  width: number,
+  height: number
+];
+
+type FormattingFunction = (value: number) => number;
+
+export interface MiniTooltipAPI {
+  hide(): void;
```
Comment (DalerAsrorov) [inline src/typings/charts/mini-tooltip.d.ts:34]: Might as well write a comment for the "hide" method

File: src/typings/charts/mini-tooltip.d.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+export type TooltipSelection = Selection<
+  BaseType,
+  MiniTooltipDataShape,
+  HTMLElement,
+  any
+>;
+
+type MousePosition = [
```
Comment (DalerAsrorov) [inline src/typings/charts/mini-tooltip.d.ts:21]: I think exporting MousePosition could be useful. What do you think about the ChartSize?

**Review by DalerAsrorov — CHANGES_REQUESTED** [review]

File: src/typings/charts/mini-tooltip.d.ts (hunk 2 above)
Comment (Golodhros) [reply src/typings/charts/mini-tooltip.d.ts:21]: Makes sense!

File: src/typings/charts/mini-tooltip.d.ts (hunk 1 above)
Comment (Golodhros) [reply src/typings/charts/mini-tooltip.d.ts:34]: Oops, it got lost somewhere!
Adding it

**Review by DalerAsrorov — APPROVED** [review]
Thanks!

**Pull request merged**